src/IO/Exporter/PointwiseInterpolator.cpp

```diff
@@ -89,8 +97,21 @@ void interpolate_to_points(
       tensor_data.push_back(std::move(double_component_data));
     }
   }
-  h5file.close();
-#pragma omp parallel num_threads(num_threads)
+  return tensor_data;
+}
+
+template <typename ResultDataType, size_t Dim>
+void _interpolate_to_points(
```

Comment:
Why the underscores at the beginning of the function names? I thought our convention with functions in cpp files was just an anonymous namespace

Reply:
changed underscore to `_impl`